docs/UserGuide.adoc

```diff
@@ -62,7 +62,7 @@ Format: `help`
 
 === Adding a person: `add`
 
```

Comment:
The find and replace changed Address Book into matricNumber book 😅 We'll eventually refactor our code to a 'Club book' or something like that